src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setColor(colorSupplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
let's make it a class constant (use CONSTANT_CASE, like MAX_BOUND). Let's change it too 101, because value is exclusive (up to 100, but not 100)

Reply:
not fixed